diff --git a/src/Sql/SqlMysql.php b/src/Sql/SqlMysql.php
--- a/src/Sql/SqlMysql.php
+++ b/src/Sql/SqlMysql.php
@@ -119,6 +119,8 @@ class SqlMysql extends SqlBase
             $attribs['ssl_verify_server_cert'] = Pdo\Mysql::ATTR_SSL_VERIFY_SERVER_CERT;
         } elseif (defined('PDO::MYSQL_ATTR_SSL_VERIFY_SERVER_CERT')) {
             $attribs['ssl_verify_server_cert'] = PDO::MYSQL_ATTR_SSL_VERIFY_SERVER_CERT;
+        } else {
+            $attribs['ssl_verify_server_cert'] = 1016;
         }
 
         if (!empty($dbSpec['pdo'][$attribs['ssl_ca']])) {
